GOVERNANCE.md

```diff
@@ -118,9 +116,10 @@ Approver status is scoped to the repo/code in the Tinkerbell org.
 
 The following applies to the repo or part of the codebase for which one would be a Approver in an OWNERS file.
 
-- Reviewer for at least 3 months
-- Primary reviewer for at least 5 PRs to the codebase
-- Reviewed or merged at least 20 substantial PRs to the codebase
+- Reviewer for at least 1 month
+- Shallow understanding of the technical goals and direction of the repository
+- Shallow understanding of the technical domain of the repository
+- Reviewed or merged at least 3 substantial PRs to the codebase such as significant re-design or whole new features
```

Comment:
The 'Primary reviewer' point has been removed given you need to be a Reviewer - which stipulates that requirement - before hand. Do we want a 'Primary reviewer on an _additional_ 3 PRs'?

Reply:
We might also consider, given the lack of 'substantial changes' reducing this to just 1 or 2.

Reply:
I'd be alright with reducing "reviewed / merged" to just 1